v63004/u_inref/gtm8909.csh

```diff
@@ -23,7 +23,6 @@ mv expect.out expect.outx	# move .out to .outx to avoid -E- from being caught by
 perl $gtm_tst/com/expectsanitize.pl expect.outx > expect_sanitized.outx
 # The output is variable on slow vs fast systems and so filter out just the essential part of it to keep it deterministic.
 
-
-cat expect_sanitized.outx
+$grep -P "\S" expect_sanitized.outx
```

Comment:
grep -P man pages say this is highly experimental (that said, knowing grep can use perl type regexp searching is a good find). So use instead $grep -v '^$'. This is a simple way to filter out empty lines. Also add a comment before this line indicating we have seen extra empty lines in rare cases of test runs on slower systems (and don't have a precise understanding of the cause) and since they are not essential, we are filtering those out.

Reply:
grep -v '^$' has now been implemented with an appropriate comment